docs/tutorials/getting_started/moran.rst

```diff
@@ -78,14 +77,33 @@ The scores in each round::
     [7.0, 7.0, 7.0, 0.0]
     [7.0, 7.0, 7.0, 0.0]
 
+We can plot the results of a Moran process with `mp.populations_plot()`. Let's
+use a larger population to get a bit more data::
+
+   >>> import random
+   >>> import matplotlib.pyplot as plt
+   >>> axl.seed(15)  # for reproducible example
+   >>> N = 10
+   >>> players = [random.choice(axl.basic_strategies)() for i in range(N)]
```

Comment:
Could we hard code a list of strategies please. Otherwise, if and when `basic_strategies` changes (with new strategies being added) this would not necessarily give the same plot.